akka-http-core/src/main/scala/akka/http/javadsl/Http.scala

```diff
@@ -330,7 +333,6 @@ class Http(system: ExtendedActorSystem) extends akka.actor.Extension {
    * for encryption on the connection.
    */
   def newHostConnectionPoolTls[T](host: String, port: Int,
-                                  options: JIterable[Inet.SocketOption],
```

Comment:
No, at some point I moved the options into the settings but it seems I forgot to remove all the parameters I had added before.